src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/ApplicationClosingDateValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Esfa.Vacancy.Domain.Validation;
+using FluentValidation;
+
+namespace Esfa.Vacancy.Application.Commands.CreateApprenticeship.Validators
+{
+    public class ApplicationClosingDateValidator : AbstractValidator<DateTime>
+    {
+        public ApplicationClosingDateValidator()
+        {
+            RuleFor(closingDate => closingDate)
+                .GreaterThanOrEqualTo(DateTime.Today.AddDays(1))
```

Comment:
Might as well move this in to the main validator as this is pretty straight forward.

Reply:
The validation in RAA is that Closing Date should be at least 2 weeks from current date. In Manage it is a advisory only. I think we will have to apply the rules specified in RAA rather than MA.

Reply:
I think it is a suggestion about the two weeks later thing - https://github.com/SkillsFundingAgency/FindApprenticeship/blob/7ff41a0e656129a4ebdf3edf01c5303dda63e7f7/src/SFA.Apprenticeships.Domain.Entities.Raa/Vacancies/Constants/VacancyMessages.cs#L303

Reply:
^^ Ignore, I got confused.